docs/markdown/development.md

```diff
@@ -10,12 +10,11 @@ component: false
 - **Install node**
```

Comment:
missing links here

also can we remove bold style from links, and only show the underline?

Reply:
Done!